### PR TITLE
[PBW-4495] adding loadingImage to skin.json and removing share screen 'share' part

### PR DESCRIPTION
**Pull request by ksunika**

### CODE REVIEW

File: skin.json [hunk 1]
```diff
@@ -1,6 +1,7 @@
 {
   "general": {
-    "watermark": {"imageResource": {"url": "assets/images/ooyala-watermark.png" } }
+    "watermark": {"imageResource": {"url": "assets/images/ooyala-watermark.png" } },
+    "loadingImage": {"imageResource": {"url": "assets/images/loader_svg.svg" } }
   },
```
Comment (chibzu) [inline skin.json]: Do we want to switch these to absolute while we're at it?

File: skin.json [hunk 2]
```diff
@@ -89,24 +90,6 @@
...
-      },
-      {
-        "buttonName": "Twitter",
-        "icon": {
-          "fontFamilyName": "fontawesome",
-          "fontString": "\uf099",
-          "fontStyleClass": "icon icon-twitter"
-        }
-      }
-    ],
```
Comment (chibzu) [inline skin.json]: I think PBA had been working with these, do we have their signoff to remove them?

Comment (ksunika) [inline skin.json]: Yes, I consulted with Mike, they don't use these

File: skin.json (hunk 1 above)
Comment (ksunika) [inline skin.json]: I wanted to do all links at once converted to absolute in a different PR

File: skin.json (hunk 2 above)
Comment (lental) [inline skin.json]: signed off

**Comment by chibzu** [comment]
Great.

**Pull request merged**